packages/teams-js/src/public/app.ts

```diff
@@ -732,8 +737,8 @@ export namespace app {
     GlobalVars.initializeCompleted = false;
     GlobalVars.initializePromise = null;
     GlobalVars.additionalValidOrigins = [];
-    GlobalVars.frameContext = null;
-    GlobalVars.hostClientType = null;
+    GlobalVars.frameContext = undefined;
+    GlobalVars.hostClientType = undefined;
```

Comment:
These two were already possibly `undefined` in some cases, so I updated this code to use `undefined` as the uninitialized indicator so that there's only one way that's used to indicate "uninitialized"